docs/configuration/openconfig.md

```diff
@@ -156,6 +156,110 @@ provider smash
 provider sysdb
 ```
 
+## Syslog messages with OpenConfig
```

Comment:
It might be more suitable to call this `## AAA Accounting for gNMI/OpenConfig` as you can send the accounting logs to tacacs+, radius and syslog too

Reply:
done